### ISSUE TITLE
BUILD_SHARED option in SERIAL_RELEASE.config

### ISSUE THREAD

**Issue by d-meiser** [trilinos/Trilinos#249]
The configuration templates produced by the `checkin-test.py` script (e.g. `SERIAL_RELEASE.config`) contain the option `-DBUILD_SHARED:BOOL=ON` option. I can't find that option in the cmake directory:

```
[dmeiser@ivyamd cmake]$ pwd
/scr_ivyamd/dmeiser/Trilinos/cmake

[dmeiser@ivyamd cmake]$ grep -R BUILD_SHARED * | grep -v BUILD_SHARED_LIBS
TODO:- Enable BUILD_SHARED=ON if Trilinos_ENABLE_PyTrilinos=ON if BUILD_SHARED is
TODO:- Report an error if Trilinos_ENABLE_PyTrilinos=ON but BUILD_SHARED=OFF and
Binary file tribits/ci_support/CheckinTest.pyc matches
tribits/ci_support/CheckinTest.py:      "#-DBUILD_SHARED:BOOL=ON\n" \
tribits/ci_support/CheckinTest.py:      "#-DBUILD_SHARED:BOOL=ON\n"
```

Is this supposed  to be `-DBUILD_SHARED_LIBS:BOOL=ON`?

**Comment by bartlettroscoe** [comment]
I fixed this in TriBITSPub/TriBITS#155.  The next time I sync TriBITS to Trilinos, this will be fixed.

**Comment by bartlettroscoe** [comment]
This is fixed.  Closing as complete.
